apps/apple/FluentUITester/ios/FluentUITester.xcodeproj/project.pbxproj

```diff
@@ -283,29 +252,139 @@
 /* Begin XCBuildConfiguration section */
 		13B07F941A680F5B00A75B9A /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 9F3A29F4244F657400A401D2 /* Debug.xcconfig */;
+			baseConfigurationReference = 42B17889AC4A7BFCDD352339 /* Pods-FluentUITester.debug.xcconfig */;
 			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CURRENT_PROJECT_VERSION = 1;
+				DEAD_CODE_STRIPPING = NO;
+				INFOPLIST_FILE = FluentUITester/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				OTHER_LDFLAGS = (
+					"$(inherited)",
+					"-ObjC",
+					"-lc++",
+				);
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_NAME = FluentUITester;
+				VERSIONING_SYSTEM = "apple-generic";
```

Comment:
Should avoid any build settings in the pbxproj. These add considerable complexity as they are duplicated per target. This is what needs to be broken out to xcconfigs

Reply:
Can go to the build settings pane in the xcodeproj, CMD+A, delete to easily do this.